src/background/events.ts

```diff
@@ -30,6 +30,7 @@ export const emitter = createNanoEvents<{
   accountsChanged: () => void;
   chainsUpdated: () => void;
   chainChanged: (chain: Chain) => void;
+  registerChain: (chain: Chain) => void;
```

Comment:
Events names should describe something that has happened, not prescribe what should happen